docs/source/science/ESA_CCI/ESA_CCI_V5_Token_Access.ipynb

```diff
@@ -0,0 +1,379 @@
+{
```

Comment:
<p>This example should come first. Using the requests library should be adavnced notes. Also httpx is the favored library these days.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/514/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>